backend/curriculum_tracking/api_views.py

```diff
@@ -387,8 +387,14 @@ def cancel_review_request(self, request, pk=None):
         elif card.topic_progress:
             card.topic_progress.cancel_request_review()
 
-        # card.status = models.AgileCard.IN_PROGRESS
-        # card.save()
+        log_creators.log_card_review_request_cancelled(
+            card=card, actor_user=request.user
+        )
+        card.refresh_from_db()
+
+        assert (
+            card.status == models.AgileCard.IN_PROGRESS
```

Comment:
Are you sure? 
If a card is in review feedback then a review is requested, then the request is cancelled, then where should the card be? 
If it ends up in progress then I'm not sure that this is the correct behavior. We shouldn't make it seem like it's the intended behavior.

Reply:
what is funny, is that I didn't write that particular bit of code lol

Reply:
I mean I wrote this part, but not the rest:

log_creators.log_card_review_request_cancelled(
             card=card, actor_user=request.user
         )

Reply:
oh goodness, maybe I did write it all, pattern matching from the function above. The sleep deprivation is getting to me. Trying to think how to address though, because either it should be in progress or back in review feedback, but maybe here is not the place to try assert this and instead we should write a test about this behaviour....

Reply:
ok, have added a new test - but possibly this is tested elsewhere already.....

Reply:
It looks like there two scenarios:
1. Card is in Progress, I think its ready for review; request a review, notice a bug, and cancel my review request. Card goes back to In Progress.
2. I've already gotten some feedback so my car is in Review Feedback, I think its ready for review; request a review, notice a bug, and cancel my review request. Card goes back to Review Feedback.  

@Samantha-Hampton I believe this is the current behaviour. So what we would need is a test that checks these scenarios to make sure that we're logging the right thing.